### PR TITLE
Problem_4 First Commit

### PR DESCRIPTION
**Pull request by shashisingh12**

### CODE REVIEW

File: Assignment_2/Code/problem_4.Rmd [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    m=parm[1]+parm[2]*Insurance$Holders[i]
+    l=l+dnorm(Insurance$Claims[i], mean=m, sd=exp(parm[3]), log=T) }
+  
+  return (-l)
+}
+
+
+
+p_A=optim(c(0,0,1),modelA)
+cat("MLE : b0:", p_A$par[1], "b1:", p_A$par[2], "b3:", exp(p_A$par[3]), " ")
```
Comment (Sanjay7549) [inline Assignment_2/Code/problem_4.Rmd]: b3 should be sigma here

File: Assignment_2/Code/problem_4.Rmd [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+   
+     }
+  
+  return (-l)
+}
+
+
+
+p_B=optim(c(0,0,1),modelB)
+cat("MLE : b0:", p_B$par[1], "b1:", p_B$par[2], "b3:", exp(p_B$par[3]), " ")
```
Comment (Sanjay7549) [inline Assignment_2/Code/problem_4.Rmd]: again sigma in place of b3 in output

File: Assignment_2/Code/problem_4.Rmd [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+   l=l+dlnorm(Insurance1$Claims[i], meanlog=mu, sdlog= exp(parm[3]),log=T)
+   
+     }
+  
+  return (-l)
+}
+
+
+p_C=optim(c(2,2,1),modelc)
+cat("MLE : b0:", p_C$par[1], "b1:", p_C$par[2], "b3:", exp(p_C$par[3]), " ")
```
Comment (Sanjay7549) [inline Assignment_2/Code/problem_4.Rmd]: should have been alpha beta sigma in place of bo b1 and b3

**Review by Sanjay7549 — APPROVED** [review]
I ran the code and found the result satisfactory.

**Review by aabhishekmishra2 — APPROVED** [review]
The code is working as it should be.

**Pull request merged**